client/my-sites/customer-home/free-photo-library-card/style.scss

```diff
@@ -18,9 +18,20 @@
 .free-photo-library-card__demonstration-video {
 	max-height: 600px;
 	max-width: 800px;
-	width: 100%;
-	
-	&:focus {
-		outline: none;
+	width: calc( 90vw - 16px * 2 );
```

Comment:
For folks reviewing:
- `90vw`: [Dialog width](https://github.com/Automattic/wp-calypso/blob/146f5ae230fb3e966fd9d53949a55447390b4c8b/packages/components/src/dialog/style.scss#L35).
- `16px`:  [Dialog padding in mobile screens](https://github.com/Automattic/wp-calypso/blob/146f5ae230fb3e966fd9d53949a55447390b4c8b/packages/components/src/dialog/style.scss#L44).